deployments/charmed-magma-srsRAN/main.tf

```diff
@@ -0,0 +1,925 @@
+    name     = juju_application.orc8r-prometheus-cache.name
+    endpoint = "metrics-endpoint"
+  }
+}
+
+resource "juju_integration" "orc8r-smsd_postgresql-k8s" {
+  model = juju_model.orchestrator.name
+  application {
+    name     = juju_application.orc8r-smsd.name
+    endpoint = "db"
+  }
+  application {
+    name     = juju_application.postgresql-k8s.name
+    endpoint = "db"
+  }
+}
+
+resource "juju_integration" "orc8r-state_postgresql-k8s" {
+  model = juju_model.orchestrator.name
+  application {
+    name     = juju_application.orc8r-state.name
+    endpoint = "db"
+  }
+  application {
+    name     = juju_application.postgresql-k8s.name
+    endpoint = "db"
+  }
+}
+
+resource "juju_integration" "orc8r-subscriberdb-cache_postgresql-k8s" {
+  model = juju_model.orchestrator.name
+  application {
+    name     = juju_application.orc8r-subscriberdb-cache.name
+    endpoint = "db"
+  }
+  application {
+    name     = juju_application.postgresql-k8s.name
+    endpoint = "db"
+  }
+}
+
+resource "juju_integration" "orc8r-subscriberdb_postgresql-k8s" {
+  model = juju_model.orchestrator.name
+  application {
+    name     = juju_application.orc8r-subscriberdb.name
+    endpoint = "db"
+  }
+  application {
+    name     = juju_application.postgresql-k8s.name
+    endpoint = "db"
+  }
+}
+
+resource "juju_integration" "orc8r-tenants_postgresql-k8s" {
+  model = juju_model.orchestrator.name
+  application {
+    name     = juju_application.orc8r-tenants.name
+    endpoint = "db"
+  }
+  application {
+    name     = juju_application.postgresql-k8s.name
+    endpoint = "db"
+  }
+}
+
+resource "juju_integration" "orc8r-user-grafana_orc8r-prometheus" {
+  model = juju_model.orchestrator.name
+  application {
+    name     = juju_application.orc8r-user-grafana.name
+    endpoint = "grafana-source"
+  }
+  application {
+    name     = juju_application.orc8r-prometheus.name
+    endpoint = "grafana-source"
+  }
+}
+
+resource "juju_integration" "orc8r-user-grafana_nms-magmalte" {
+  model = juju_model.orchestrator.name
+  application {
+    name     = juju_application.orc8r-user-grafana.name
+    endpoint = "grafana-auth"
+  }
+  application {
+    name     = juju_application.nms-magmalte.name
+    endpoint = "grafana-auth"
+  }
+}
+
+################################################################################
+######################## MACHINE CHARMS ########################################
+################################################################################
+
+resource "juju_model" "agw-srsran" {
+  name = "agw-srsran"
+  cloud {
+    name   = var.agw-srsran_cloud_name
+  }
+}
+
+resource "juju_application" "agw" {
+	name = "access-gateway"
+	model = juju_model.agw-srsran.name
+	charm {
+		name = "magma-access-gateway-operator"
+		channel = "beta"
+	}
+}
+
+resource "juju_application" "srsran" {
+	name = "srsran"
+	model = juju_model.agw-srsran.name
+	charm {
+		name = "charmed-osm-srs-enb-ue"
+		channel = "edge"
+	}
+}
+
```

Comment:
Are we missing the integrations between AGW and orc8r, and between AGW and srsran?